code/datums/components/storage/concrete/pockets.dm

```diff
@@ -59,27 +59,30 @@
 	silent = TRUE
 
 /datum/component/storage/concrete/pockets/shoes/Initialize()
-	. = ..()
-	set_holdable(list(
-		/obj/item/kitchen/knife, /obj/item/switchblade, /obj/item/pen,
-		/obj/item/scalpel, /obj/item/reagent_containers/syringe, /obj/item/dnainjector,
-		/obj/item/reagent_containers/hypospray/medipen, /obj/item/reagent_containers/dropper,
-		/obj/item/implanter, /obj/item/screwdriver, /obj/item/weldingtool/mini,
-		/obj/item/firing_pin
-		),
-		list(/obj/item/screwdriver/power)
-		)
+    . = ..()
+    set_holdable(list(
+        /obj/item/kitchen/knife, /obj/item/switchblade, /obj/item/pen,
+        /obj/item/scalpel, /obj/item/reagent_containers/syringe, /obj/item/dnainjector,
+        /obj/item/reagent_containers/hypospray/medipen, /obj/item/reagent_containers/dropper,
+        /obj/item/implanter, /obj/item/screwdriver, /obj/item/weldingtool/mini,
+        /obj/item/firing_pin, /obj/item/suppressor, /obj/item/ammo_box/magazine/m9mm, /obj/item/ammo_box/magazine/m45,
+        /obj/item/ammo_casing, /obj/item/lipstick, /obj/item/clothing/mask/cigarette, /obj/item/lighter,
+        /obj/item/match, /obj/item/holochip, /obj/item/toy/crayon),
+        list(/obj/item/screwdriver/power, /obj/item/ammo_casing/caseless/rocket, /obj/item/clothing/mask/cigarette/pipe, /obj/item/toy/crayon/spraycan)
+        )
 
 /datum/component/storage/concrete/pockets/shoes/clown/Initialize()
-	. = ..()
-	set_holdable(list(
-		/obj/item/kitchen/knife, /obj/item/switchblade, /obj/item/pen,
-		/obj/item/scalpel, /obj/item/reagent_containers/syringe, /obj/item/dnainjector,
-		/obj/item/reagent_containers/hypospray/medipen, /obj/item/reagent_containers/dropper,
-		/obj/item/implanter, /obj/item/screwdriver, /obj/item/weldingtool/mini,
-		/obj/item/firing_pin, /obj/item/bikehorn),
-		list(/obj/item/screwdriver/power)
-		)
+    . = ..()
+    set_holdable(list(
+        /obj/item/kitchen/knife, /obj/item/switchblade, /obj/item/pen,
+        /obj/item/scalpel, /obj/item/reagent_containers/syringe, /obj/item/dnainjector,
+        /obj/item/reagent_containers/hypospray/medipen, /obj/item/reagent_containers/dropper,
+        /obj/item/implanter, /obj/item/screwdriver, /obj/item/weldingtool/mini,
+        /obj/item/firing_pin, /obj/item/suppressor, /obj/item/ammo_box/magazine/m9mm, /obj/item/ammo_box/magazine/m45,
+        /obj/item/ammo_casing, /obj/item/lipstick, /obj/item/clothing/mask/cigarette, /obj/item/lighter,
+        /obj/item/match, /obj/item/holochip, /obj/item/toy/crayon, /obj/item/bikehorn),
+        list(/obj/item/screwdriver/power, /obj/item/ammo_casing/caseless/rocket, /obj/item/clothing/mask/cigarette/pipe, /obj/item/toy/crayon/spraycan)
+        )
```

Comment:
Put a new line at the end of each item, makes things less cluttered then 3 wide.
It also makes figuring out what you're adding here a bit easier, though the code initially had this issue too.